emc2/simulator/radar_moments.py

```diff
@@ -593,107 +599,111 @@ def calc_radar_micro(instrument, model, z_values, atm_ext, OD_from_sfc=True,
         model.ds["sub_col_sigma_d_%s_strat" % hyd_type].attrs["Processing method"] = method_str
     model.ds["sub_col_Vd_tot_strat"] = xr.DataArray(V_d_numer_tot / moment_denom_tot,
                                                     dims=model.ds["sub_col_Ze_tot_strat"].dims)
-    print("Now calculating total spectral width (this may take some time)")
-    for hyd_type in hyd_types:
-        fits_ds = calc_mu_lambda(model, hyd_type, subcolumns=True, **kwargs).ds
-        N_0 = fits_ds["N_0"].values
-        lambdas = fits_ds["lambda"].values
-        mu = fits_ds["mu"].values
-
-        if np.isin(hyd_type, optional_ice_classes):
-            if mie_for_ice:
-                if hyd_type == "ci":
-                    hyd_type_2_use = "ci"
+    if skip_spectral_width:
```

Comment:
All `skip_spectral_width` instances except for this one are `not` statements, so I think it would be more intuitive to change that input parameter to `calc_spectral_width` or something of some sort with a default `True` value and then switch between the if-else options here.